frontend/src/app/AppLauncher.tsx

```diff
@@ -20,7 +20,7 @@ type ApplicationAction = {
 };
 
 type Section = {
-  label: string;
+  label?: string;
```

Comment:
Why are you marking this as optional? According to the type, label is based on `text` which is a required attribute, don't make optional attributes that are required.

Reply:
Also, we tend to separate types from components, is it really necessary that we keep this type here?

Reply:
Also, do you want me to move the `Section` type to `k8sTypes.ts` or `types.ts`?

Reply:
Also, the `K8sResourceCommon` is different in *backend* and *frontend*, should that be changed? Also, making the ConsoleLinkKind type same in *backend* and *frontend*